uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.controller.js

```diff
@@ -200,7 +200,9 @@ define([
 
         $scope.$broadcast('ct-spinner-show');
 
-        documentStatus = vm.isRole('admin') ? '3' : (vm.files.length || vm.uploader.queue.length ? '2' : '1');
+        documentStatus = vm.isRole('admin') ?
```

Comment:
A function can be created to abstract this, `initDocumentStat()` which will abstract this logic. That will make it more readable.